Formula/sqlc.rb

```diff
@@ -0,0 +1,40 @@
+class Sqlc < Formula
+  desc "Generate type safe Go from SQL"
+  homepage "https://sqlc.dev/"
+  url "https://github.com/kyleconroy/sqlc/archive/v1.6.0.tar.gz"
+  sha256 "a95a123f29a71f5a2eea0811e4590d59cbc92eccd407abe93c110c738fc4740b"
+  license "MIT"
+  head "https://github.com/kyleconroy/sqlc.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "./cmd/sqlc"
+  end
+
+  test do
+    (testpath/"sqlc.json").write <<~SQLC
+      {
+        "version": "1",
+        "packages": [
+          {
+            "name": "db",
+            "path": ".",
+            "queries": "query.sql",
+            "schema": "query.sql",
+            "engine": "postgresql"
+          }
+        ]
+      }
+    SQLC
+
+    (testpath/"query.sql").write <<~SQL.squish
+      CREATE TABLE foo (bar text);
+
+      -- name: SelectFoo :many
+      SELECT * FROM foo;
+    SQL
+
+    system bin/"sqlc", "generate"
```

Comment:
it would be nice to assert the response like other go formulae are doing.

Reply:
@chenrui333 do you have an example I can look at?

Reply:
@kyleconroy See the `mysql` test block, for instance. Relevant docs:

* https://www.rubydoc.info/github/test-unit/test-unit/Test/Unit/Assertions
* https://rubydoc.brew.sh/Homebrew/Assertions.html#shell_output-instance_method

Reply:
I wrote some tests, should be good to go now :)

Reply:
Can you fix the one lint error?

Reply:
looks good now. :)

Reply:
Thanks!